python/tests/stress/arcticdb/version_store/test_stress_multicolumn.py

```diff
@@ -12,7 +12,7 @@
 import pytest
 
 from arcticdb.util.test import assert_frame_equal
-from arcticdb_ext.tools import AZURE_SUPPORT
+from arcticdb.util._versions import IS_PANDAS_TWO
```

Comment:
Not used?

Reply:
I guess I need to rebase this branch. One moment

Reply:
Done